eclair-core/src/test/scala/fr/acinq/eclair/router/GraphSpec.scala

```diff
@@ -0,0 +1,220 @@
+package fr.acinq.eclair.router
+
+import fr.acinq.bitcoin.Crypto.PublicKey
+import org.scalatest.FunSuite
+import RouteCalculationSpec._
+import fr.acinq.eclair.ShortChannelId
+import fr.acinq.eclair.router.Graph.GraphStructure.{GraphEdge, DirectedGraph}
+import fr.acinq.eclair.wire.ChannelUpdate
+
+class GraphSpec extends FunSuite {
```

Comment:
Indentation in this file is not consistent with the rest of the project, 2 spaces should be used, not tabs.